components/dsl/resources/ome/dsl/object.vm

```diff
@@ -319,6 +319,15 @@ implements java.io.Serializable, IObject
     */
     private static final long serialVersionUID = 0000000030000020302L;
 
+#if($type.isEnum && !$type.shortname.startsWith("Units") && ${type.shortname} != "Binning" && ${type.shortname} != "PixelsType")
```

Comment:
They have values that wouldn't make good variable names if we rethought the `VALUE_` prefix. I thought better not to provide them for now if they might be more deeply rethought later as they can easily be added in the unlikely event of anybody actually missing them. (Their values *are* already available to code on the Blitz code-generated side.)

Yeah, the codegen for the Blitz side does a bit of fiddling with the units, some of which is unnecessary from the Java point of view, and again I was trying to defer tricky decisions that we might later regret and just include the easy stuff while that already provides plenty of value added (at least from my point of view).